jungchaeeun/12주차/프로그래머스 12940번 문제.py

```diff
@@ -0,0 +1,26 @@
+'''
+문제 설명
+두 수를 입력받아 두 수의 최대공약수와 최소공배수를 반환하는 함수, solution을 완성해 보세요. 
+배열의 맨 앞에 최대공약수, 그다음 최소공배수를 넣어 반환하면 됩니다. 
+예를 들어 두 수 3, 12의 최대공약수는 3, 최소공배수는 12이므로 solution(3, 12)는 [3, 12]를 반환해야 합니다.
+
+제한 사항
+두 수는 1이상 1000000이하의 자연수입니다.
+
+입출력 예
+n	    m	    return
+3	   12	   [3, 12]
+2	    5	   [1, 10]
+'''
+
+
+def gcd(a, b):
```

Comment:
```suggestion
import math
math.gcd(a, b)
```
math 라이브러리에 주어지는 gcd 함수를 이용해보면 더욱 편리하게 코드 작성할수 있습니다